src/scancode/api.py

```diff
@@ -239,6 +239,7 @@ def _licenses_data_from_match(
         result['short_name'] = lic.short_name
         result['category'] = lic.category
         result['is_exception'] = lic.is_exception
+        result['is_unknown']=lic.is_unknown
```

Comment:
The spacing needs some fixing:
```suggestion
        result['is_unknown'] = lic.is_unknown
```